integrations/server/test_cdc.py

```diff
@@ -0,0 +1,43 @@
+# first party
+from delphi.epidata.common.integration_test_base_class import BasicIntegrationTest
+
+
+class CdcTest(BasicIntegrationTest):
+    """Basic integration tests for cdc endpint."""
+
+    def setUp(self) -> None:
+        self.truncate_tables_list = ["cdc_extract"]
+        self.role_name = "cdc"
+        super().setUp()
```

Comment:
if you follow a pattern similar to the [`localSetUp()` used in in `CovidcastBase`](https://github.com/cmu-delphi/delphi-epidata/blob/f7da6598a810be8df5374e3a71512c631c3a14f1/src/acquisition/covidcast/test_utils.py#L168), you can avoid having to call `super()`

i believe you should also be able to set `truncate_tables_list` and `role_name` as class variables (outside of the class method scopes) to achieve the same effect, if you would rather try that instead

Reply:
Nice!